Sources/BraintreeCore/BTPostalAddress.swift

```diff
@@ -37,4 +53,33 @@ import Foundation
         case recipientName = "recipient_name"
         case streetAddress = "line1"
     }
+    
+    // MARK: - Helper Method
+    
+    /// Returns address components as a dictionary for accessing internal properties across modules
+    public func addressComponents() -> [String: String] {
+        var components: [String: String] = [:]
+        if let recipientName = recipientName {
```

Comment:
nit, for all of these we can do:

```suggestion
        if let recipientName {
```